scripts/hostapd/certs/ca.cnf

```diff
@@ -0,0 +1,59 @@
+[ ca ]
+default_ca		= CA_default
+
+[ CA_default ]
+dir			= ./
+certs			= $dir
+crl_dir			= $dir/crl
+database		= $dir/index.txt
+new_certs_dir		= $dir
+certificate		= $dir/ca.pem
+serial			= $dir/serial
+crl			= $dir/crl.pem
+private_key		= $dir/ca.key
+RANDFILE		= $dir/.rand
+name_opt		= ca_default
+cert_opt		= ca_default
+default_days		= 365
+default_crl_days	= 30
+default_md		= md5
```

Comment:
Yea I am just worried about compatibility issues: https://github.com/tomeshnet/prototype-cjdns-pi2/pull/63/files#diff-cc31f3ada4ea58c3308ada2613b24837R213 given this is a prototype repo I think sha1 is sufficient?